checkov/secrets/parsers/terraform/multiline_parser.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+import re
+from typing import TYPE_CHECKING
+
+from checkov.common.parsers.multiline_parser import BaseMultiLineParser
+
+if TYPE_CHECKING:
+    from detect_secrets.util.code_snippet import CodeSnippet
+
+START_OBJ = re.compile(r"\s*\w*\s*=?\s*{")
+END_OBJ = re.compile(r"\s*}")
+COMMENT_PREFIX = re.compile(r"^[\s]*(#|\/\/)")
+
+
+class TerraformMultiLineParser(BaseMultiLineParser):
+    def consecutive_lines_in_same_object(
+        self,
+        raw_context: CodeSnippet | None,
+        other_line_idx: int,
+    ) -> bool:
+        if not raw_context or not 0 <= other_line_idx < len(raw_context.lines):
+            return False
+
+        return True
```

Comment:
```suggestion
        return raw_context and 0 <= other_line_idx < len(raw_context.lines)
```

Reply:
this happens, when you write your code several times and forget to simplify it, after removing a lot of code 😄